<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWidget</class>
 <widget class="QWidget" name="ServerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServerWidget</string>
  </property>
  <widget class="QPushButton" name="startStopServerButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>361</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Stoped</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectDirectoryButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="selectedDirectoryPath">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>311</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="fileTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>21</y>
     <width>361</width>
     <height>201</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <column>
    <property name="text">
     <string>Files</string>
    </property>
    <property name="textAlignment">
     <set>AlignHCenter|AlignVCenter|AlignCenter</set>
    </property>
   </column>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
